modules/installation-configuration-parameters.adoc

```diff
@@ -449,7 +449,7 @@ Optional installation configuration parameters are described in the following ta
 |String array
 
 |`capabilities.baselineCapabilitySet`
-|Selects an initial set of optional capabilities to enable. Valid values are `None`, `v4.11` and `vCurrent`. `v4.11` enables the `baremetal` Operator, the `marketplace` Operator, and the `openshift-samples` content. `vCurrent` installs the recommended set of capabilities for the current version of {product-title}. The default value is `vCurrent`.
+|Selects an initial set of optional capabilities to enable. Valid values are `None`, `v4.11` and `vCurrent`. `v4.11` enables the `baremetal` Operator, the `marketplace` Operator, and the `openshift-samples` content. `vCurrent` installs the recommended set of capabilities for the current version of {product-title}. The default value is `vCurrent`. The `marketplace` Operator populates OperatorHub with a set of default Operator catalogs. If you disable the `marketplace` Operator, you can install Operators by manually creating a catalog source to populate OperatorHub. {product-title} does not automatically update manually created catalog sources.
```

Comment:
i wouldn't put this sentence here, i'd put it somewhere that's specifically describing the implication of enabling/disabling the marketplace capability, such as what i tried to do in my original PR (breaking out explicit descriptions of what enabling/disabling each cap does)

Reply:
@bparees, I think we need to say something about the larger implications. Would this be acceptable instead?
> If you disable the `marketplace` Operator, you must take additional steps to populate OperatorHub.

Reply:
i wasn't disagreeing w/ stating the implications *somewhere*.  i'm disagreeing with where we should put the implication.

What is going to happen when we have 15 caps that all have their own implications for being disabled?  Put them all here in one paragraph?

it's also not very eye catching or easy to find for someone who explicitly wants to go read about what happens if they disable a particular cap.

that is why I created my original PR that broke out each cap so you could specifically describe the implications of disabling any particular capability, and people can more easily see/find the implications associated w/ each capability.

my PR for reference: https://github.com/openshift/openshift-docs/pull/50771

Reply:
Reworded to simply say "If you disable the `marketplace` Operator, you must take additional steps to populate OperatorHub." I would like to give some heads up that the Operator interacts with OperatorHub in some way. WDYT?

Reply:
if you want to say anything here, you should say something like "disabling any particular capability has implications for cluster functionality.  See below for the specific implications of disabling a particular capability"

(and then below, have those implications broken out by capability)

Reply:
> Reworded to simply say "If you disable the marketplace Operator, you must take additional steps to populate OperatorHub." I would like to give some heads up that the Operator interacts with OperatorHub in some way. WDYT?

i still think this is not the place to talk about the implications of disabling any one specific capability.  It would be appropriate to give a generic warning that disabling capabilities has implications, and then direct them to the more detailed documentation (which needs to be written) where those implications are discussed at length.

Putting the implications here doesn't make sense to me for a few reasons, but the most obvious one is that it doesn't scale as you will need to discuss the implications of disabling many caps, not just marketplace.

Reply:
>  I agree that this is not an ideal solution

Then why can't we just use the solution i proposed and drafted?  what's wrong w/ my suggested structuring of this information?

> In 4.11, are there additional known implications besides the marketplace/OperatorHub link? 

Every capability that is disabled has implications.  If it didn't have implications, it wouldn't provide value in the product in the first place.  Disabling the samples operator means you have no s2i builder imagestreams by default and no sample templates.  Disabling the baremetal operator means you can't manage baremetal machines.

And it's just as important, imho, to discuss why you would want to enable the cap(what it provides), as why you might not want to disable it, which again is why my original PR was structured to give proper space to the discussion of what each cap does/provides and what happens if you disable it.

Reply:
Oh, we're still going to use your solution. Michael's going to get it merged when he gets back from PTO. This is an additional change, not a replacement. It's just positioned where most users aren't super likely to see it until after they've made whatever choice they've made about this flag.

It's important to try to give users the information that they need when they need it, and, to my knowledge, no one on the docs team realized that changing this flag made OperatorHub not function. I'm not confident that users will make the connection either. Because you set this parameter value here, some discussion of the implications seems appropriate, and it's better to say something multiple times to increase the odds of it being found. 

This is more of a patch in line with the scope of the capabilities feature for 4.11, and I expect it to be overhauled in  4.12 with that scope expansion.

I'd rather say a few words about the other functions here as well than not explain this one. (The others at least have the benefit of hinting at the functionality that's impacted because of the way the flags are named.)

Reply:
if this is the best we can do for now to address immediate user confusion, then i guess let's merge it.

but i think once we get to the point where we have explicit cap implications described elsewhere, we should remove this and replace it with a generic "please see X for implications about disabling specific caps".  Otherwise:

1) it seems a bit subjective which implications/caps we decide to discuss in more depth here and which we don't
2) it doesn't scale very well, as i mentioned before, to describing multiple implications for different caps

> no one on the docs team realized that changing this flag made OperatorHub not function

to be clear, disabling the marketplace does not make operatorhub not function.  It simply means you will not have any default sources of content for the operatorhub.  You can still create your own catalogsources that will show up in the operatorhub and then consume content from them, even if the marketplace is disabled.

Reply:
@bscott-rh, if you want to add a quick UL to describe the 10,000 ft view of the other flags, that would at least give them all parity. This section will also need to be reworked for 4.12.

@michaelryanpeter Please consider this thread when you revisit #[50830](https://github.com/openshift/openshift-docs/pull/50830)

@bparees You are right, and I apologize for the incorrect generalization. If you disable `marketplace` here, additional action is required from the user, which the person who filed the bug did not appear to know how to take either. :)